spec/services/rapid_ready_for_decision/hypertension_pdf_generator_spec.rb

```diff
@@ -8,6 +8,8 @@
 RSpec.describe RapidReadyForDecision::HypertensionPdfGenerator, :vcr do
   subject { PDF::Inspector::Text.analyze(pdf_generator.generate.render).strings }
 
+  let(:generated_pdf) { pdf_generator.generate }
+
```

Comment:
If not used anywhere, remove this line.
Let's add a test to check the new DOB content is actually in the pdf.